softwerkskammer/lib/mailsender/message.js

```diff
@@ -79,8 +81,8 @@ Message.prototype.toTransportObject = function (senderAddress) {
     cc: this.cc,
     bcc: this.bcc,
     subject: this.subject,
-    text: jade.renderFile(filenameTextonly, renderingOptions),
-    html: jade.renderFile(filename, renderingOptions)
+    text: jade.renderFile(filenameTextonly, renderingOptions).replace(new RegExp('\\]\\(/', 'g'), '](' +  publicUrlPrefix + '/'),
+    html: jade.renderFile(filename, renderingOptions).replace(new RegExp('<a href="/', 'g'), '<a href="' +  publicUrlPrefix + '/')
   };
```

Comment:
Wäre es nicht geschickter, am Anfang dieser Methode einmal this.markdown zu bearbeiten? Dann muss man die Ersetzung nicht zweimal machen, glaube ich.